skops/card/tests/test_card.py

```diff
@@ -26,6 +26,29 @@ def fit_model():
     return reg
 
 
+def save_model_to_file(model_instance, suffix):
+    save_file_handle, save_file = tempfile.mkstemp(suffix=suffix, prefix="skops-test")
+    if suffix in (".pkl", ".pickle"):
+        with open(save_file, "wb") as f:
+            pickle.dump(model_instance, f)
+    elif suffix == ".skops":
+        dump(model_instance, save_file)
+    return save_file_handle, save_file
+
+
+@pytest.mark.parametrize("suffix", [".pkl", ".pickle", ".skops"])
+def test_load_model(suffix):
+    model0 = LinearRegression(n_jobs=123)
+    _, save_file = save_model_to_file(model0, suffix)
+    loaded_model_str = _load_model(save_file)
+    save_file_path = Path(save_file)
+    loaded_model_path = _load_model(save_file_path)
+
+    assert loaded_model_str.n_jobs == model0.n_jobs
+    assert loaded_model_path.n_jobs == model0.n_jobs
```

Comment:
```suggestion
    assert loaded_model_str.n_jobs == 123
    assert loaded_model_path.n_jobs == 123
```

This test is a tiny bit more robust this way (in case that `model0` is being mutated by `_load_model` for some reason).